src/App.js

```diff
@@ -38,7 +38,8 @@ class App extends Component {
           occupation: 'FBI agent',
           avatar: 'https://pbs.twimg.com/profile_images/718881904834056192/WnMTb__R.jpg'
         }
-      ]
+      ],
+      searchText: ''
```

Comment:
You are calling trim [here](https://github.com/Stephrod/contact-list-starter/blob/master/src/App.js#L53) on `searchText`, but you do not have a default value set. Here, I set it to `''`